src/components/MainDashboard/MainDashboard.styled.js

```diff
@@ -9,43 +9,48 @@ export const STitle_h3 = styled.h3`
   margin-bottom: 10px;
 `;
 
-export const SButton_button = styled.button`
+export const SColumnWrapper = styled.div`
   display: flex;
-  flex-direction: row;
-  justify-content: center;
-  align-items: center;
+  gap: 14px;
+`;
 
-  gap: 8px;
-  width: 335px;
-  height: 56px;
-  background-color: ${({ theme }) => theme.mainDashbordButton};
+// export const SButton_button = styled.button`
```

Comment:
this style i don't need , bc i done it in my component, but i dont want to delete smth without your opinion